Report # events per Lumi for output files in framework job report

Dr15Jones opened the pull request:
DMWM needs to obtain the number of events written per LuminosityBlock for each output file.

Dr15Jones committed:
Report # events per Lumi for output files in framework job report

WMDM needs to obtain the number of events written per LuminosityBlock for each output file.

cmsbuild commented:
A new Pull Request was created by  @Dr15Jones (Chris Jones) for CMSSW_8_0_X.

It involves the following packages:

FWCore/MessageLogger
IOPool/Output


@cmsbuild, @smuzaffar, @Dr15Jones, @davidlange6 can you please review it and eventually sign? Thanks.
@Martin-Grunewald, @wddgit, @wmtan this is something you requested to watch as well.
@davidlange6, @smuzaffar you are the release manager for this.

cms-bot commands are listed here cms-sw/cmssw#13028

Dr15Jones commented:
backport #17912

Dr15Jones commented:
please test

Dr15Jones commented:
+1

cmsbuild commented:
The tests are being triggered in jenkins.

cmsbuild commented:
This pull request is fully signed and it will be integrated in one of the next CMSSW_8_0_X IBs after it passes the integration tests. This pull request requires discussion in the ORP meeting before it's merged. @davidlange6, @smuzaffar

Dr15Jones commented:
@ericvaandering

Dr15Jones committed:
Fix comparisons used for unit tests

Recent changes to the framework job report output broke unit tests which were comparing expected reports to actual reports.

cmsbuild commented:
Pull request #17927 was updated. @cmsbuild, @smuzaffar, @Dr15Jones, @davidlange6 can you please check and sign again.

Dr15Jones commented:
+1

Dr15Jones commented:
please test

cmsbuild commented:
The tests are being triggered in jenkins.

cmsbuild commented:
This pull request is fully signed and it will be integrated in one of the next CMSSW_8_0_X IBs after it passes the integration tests. This pull request requires discussion in the ORP meeting before it's merged. @davidlange6, @smuzaffar

cmsbuild commented:
+1
Tested at: 8c83ea8945b786f3cec99bfd0507e39f72a473c0
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-17927/18417/summary.html

cmsbuild commented:
Comparison job queued.

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-17927/18417/summary.html

cmsbuild commented:
+1
Tested at: 57f73150c8bd8b6bb897c68f9f6cc6dbbad6111c
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-17927/18432/summary.html

cmsbuild commented:
Comparison job queued.

davidlange6 commented:
+1
